components/edit-collective/sections/Host.js

```diff
@@ -226,14 +224,13 @@ class Host extends React.Component {
                   {collective.stats.balance === 0 && (
                     <Fragment>
                       <p>
-                        <Button
-                          bsStyle="primary"
-                          type="submit"
```

Comment:
It's good to keep the `type="submit"` if already set, makes things explicit in the DOM